nano/store/lmdb/lmdb.cpp

```diff
@@ -242,6 +251,35 @@ void nano::store::lmdb::component::upgrade_v21_to_v22 (store::write_transaction
 	logger.info (nano::log::type::lmdb, "Upgrading database from v21 to v22 completed");
 }
 
+// Fill rep_weights table with all existing representatives and their vote weight
+void nano::store::lmdb::component::upgrade_v22_to_v23 (store::write_transaction const & transaction_a)
```

Comment:
I was wondering if it is a problem if the write_transaction is long running?
Also should we indicate progress here? Like printing a dot after every x accounts?

Reply:
Even a basic progress indicator would be great, from a user perspective it might not be clear if the process hanged or is progressing.

Reply:
I've added a progress log statement for every 250,000 accounts